.circleci/config.yml

```diff
@@ -613,12 +638,16 @@ jobs:
     executor: integration-test-executor
     parallelism: 4
     steps:
+      - attach_workspace:
```

Comment:
Is this necessary? Add 3+ minutes.

Reply:
Hrm, maybe I just don't understand circle enough.

It was in the _command_ before, I just moved it to the _job_ entries that call that command to match the way all the other jobs are set up -- but it should have been happening either way.  (Those jobs call the command, so the attach should happen once per job whether it's defined in the job or the command, right?)

I can take it out, it's not directly related to my changes, I just ran into that inconsistency while I was working on the config.

Reply:
My mistake, let's keep it.